internal/jobs/twitter_test.go

```diff
@@ -12,53 +17,99 @@ import (
 	"github.com/masa-finance/tee-worker/api/types"
 	. "github.com/masa-finance/tee-worker/internal/jobs"
 	"github.com/masa-finance/tee-worker/internal/jobs/stats"
+	"github.com/masa-finance/tee-worker/internal/jobs/twitterx"
 )
 
+// parseTwitterAccounts parses TWITTER_ACCOUNTS environment variable like production does
+func parseTwitterAccounts() []string {
+	accountsEnv := os.Getenv("TWITTER_ACCOUNTS")
+	if accountsEnv == "" {
+		return nil
+	}
+
+	accounts := strings.Split(accountsEnv, ",")
+	for i, account := range accounts {
+		accounts[i] = strings.TrimSpace(account)
+	}
+	return accounts
+}
+
+// parseTwitterApiKeys parses TWITTER_API_KEYS environment variable like production does
+func parseTwitterApiKeys() []string {
+	apiKeysEnv := os.Getenv("TWITTER_API_KEYS")
+	if apiKeysEnv == "" {
+		return nil
```

Comment:
ditto, how about returning `[]string{}`?

Reply:
captured sim to above!